.github/workflows/cpp-packaging.yml

```diff
@@ -8,8 +8,6 @@ on:
     - cron: "0 9 * * *"  # 9am UTC = 1am PST / 2am PDT
   workflow_dispatch:
     inputs:
-      commitIdToPackage:
-        description: 'commit ID to package'
```

Comment:
Why was this taken out?  It seems useful. 

Is the intention that we would create a branch from a particular commit instead, and just run the workflow on that branch?

Reply:
Yes, because you can only trigger a workflow on a branch or tag, not on any old commit hash, I wanted to ensure that we are definitely triggering the integration_tests workflow on the same git revision as this script is running on. This removes the ambiguity. If you need to package any old commit, make a branch or tag for it first.